### PR TITLE
Contains the pipeline now as a lambda function, along with terraform

### PR DESCRIPTION
**Pull request by MilesDrabwell**
This changes the pipeline to now run as a lambda pipeline and also has terraform to set up the services needed to run on AWS.

### CODE REVIEW

**Review by HighestAuto — COMMENTED** [review]
Maybe to avoid dry, make one file that has the get_connection and import to where it is needed

**Review by HighestAuto — APPROVED** [review]

File: pipeline/pipeline.py [hunk 1]
```diff
@@ -14,28 +15,32 @@
...
+    plant_ids = new_plant_ids(plants_data)
+    logger.info("plant data acquired from api")
+    logger.info("transforming plant data ready for upload")
+    transformed = get_table_data(plants_data, conn)
+    logger.info("transformed plant data uploading")
+    load(conn, transformed)
+    logger.info("all plant data uploaded")
+    end_time = time.time()
+    duration = end_time - start_time
+    # logger.info(f"API call took {duration} seconds")
```
Comment (LinfanS) [inline pipeline/pipeline.py]: comment to be removed

File: pipeline/pipeline.py [hunk 2]
```diff
@@ -14,28 +15,32 @@
 PLANT_IDS = list(range(1, 51))
 
 
-async def main() -> None:
+async def main() -> dict:
     """Main function to run the script in a while loop"""
```
Comment (LinfanS) [inline pipeline/pipeline.py]: remember to fix docstrings

**Review by LinfanS — APPROVED** [review]
A couple smalls things to polish but general changes look good to me.

**Pull request merged**